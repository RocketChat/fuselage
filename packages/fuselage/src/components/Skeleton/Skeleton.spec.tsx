import React from 'react';
import ReactDOM from 'react-dom';

import { Skeleton } from '../..';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Skeleton />, div);
  ReactDOM.unmountComponentAtNode(div);
});
