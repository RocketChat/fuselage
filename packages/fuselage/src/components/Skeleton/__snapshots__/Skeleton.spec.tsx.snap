// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CircleVariant without crashing 1`] = `
<body>
  <div>
    <span
      class="rcx-skeleton rcx-skeleton--circle rcx-css-19saq22"
    />
  </div>
</body>
`;

exports[`renders Default without crashing 1`] = `
<body>
  <div>
    <span
      class="rcx-skeleton rcx-skeleton--text"
    />
  </div>
</body>
`;

exports[`renders RectVariant without crashing 1`] = `
<body>
  <div>
    <span
      class="rcx-skeleton rcx-skeleton--rect rcx-css-1q2xu1s"
    />
  </div>
</body>
`;

exports[`renders TextVariant without crashing 1`] = `
<body>
  <div>
    <span
      class="rcx-skeleton rcx-skeleton--text"
    />
  </div>
</body>
`;
